x-pack/plugins/security_solution/server/lib/detection_engine/scripts/roles_users/t1_analyst/index.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import * as t1AnalystUser from './detections_user.json';
+import * as t1AnalystRole from './detections_role.json';
+export { t1AnalystUser, t1AnalystRole };
```

Comment:
NOTE: I get that these are script folders, but the tests and imports cleaned up when I did the NodeJS index barrel rolls here.